src/execute.js

```diff
@@ -48,6 +54,10 @@ export function execute({
 
   const request = self.buildRequest({spec, operationId, parameters, securities, ...extras})
 
+  if (request.error) {
```

Comment:
Why are we throwing in `execute`, via the `request.error` switch, instead of throwing the error directly in `buildRequest`?